testcontainers/tests/images.rs

```diff
@@ -74,3 +88,53 @@ fn trufflesuite_ganachecli_listaccounts() {
 
     assert_that(&accounts).is_ok();
 }
+
+#[test]
+fn dynamodb_local_create_table() {
```

Comment:
Is creating a table the most simple command to check that the database is reachable? I am currently adding support for the `postgres` image and I simply execute a query to add two numbers:

https://github.com/testcontainers/testcontainers-rs/blob/a47584d8e75bb7d7fd568e689f2af9f420ccc9dd/testcontainers/tests/images.rs#L80-L98

Reply:
DynamoDB being a fairly simple "NoSQL" Db doesn't provide a lot of the niceties that you find in a full featured RDBMS like PostgreSQL. Originally, my plan was to do a ListTables command and return on successful completion. Rusoto didn't seem to like getting an empty result back as there aren't any tables yet (though maybe I should dig in a little more at some point). With that failing, the only other option was to just create a table. Creating a table is a good test anyways, as the first thing anybody is going to want to do in their test is to create a table, then perhaps populate some test data.

The reason I added a the testcontainer to begin with was to play around with AWS's new lambda support and see how easy it was to integrate rusoto. See: https://github.com/ayax79/hello_rust_lambda

Reply:
Fair enough :)

It does test that it works so I guess it's ok.